client/flutter/lib/pages/question_index.dart

```diff
@@ -36,19 +51,12 @@ class _QuestionIndexPageState extends State<QuestionIndexPage> {
       child: CustomScrollView(
         slivers: [
           SliverAppBar(
+            // Hide the built-in icon for now
             iconTheme: IconThemeData(
-              color: Colors.black,
+              color: Colors.transparent,
             ),
```

Comment:
`automaticallyImplyLeading: false` should do the same thing.

Reply:
This sounds good. We should consider for cleanup after launch.